# Generated by Django 5.0.3 on 2024-04-01 07:32

import django.utils.timezone
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('guard', '0003_requesthistory'),
    ]

    operations = [
        migrations.CreateModel(
            name='ResponseErrors',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('session_key', models.CharField(max_length=500, verbose_name='Ключ сессии')),
                ('date', models.DateTimeField(default=django.utils.timezone.now, verbose_name='Первое обращение')),
                ('ip', models.GenericIPAddressField(verbose_name='IP адрес')),
                ('user_agent', models.CharField(max_length=1000, verbose_name='User agent')),
                ('error_code', models.SmallIntegerField(verbose_name='Код ошибки')),
            ],
            options={
                'verbose_name': 'Ошибка запроса',
                'verbose_name_plural': 'Ошибки запросов',
            },
        ),
    ]
